lib/promoted/ruby/client/options.rb

```diff
@@ -0,0 +1,215 @@
+module Promoted
+  module Ruby
+    module Client
+      DELIVERY_TIMEOUT_MILLIS = 30000
+      DEFAULT_METRICS_TIMEOUT_MILLIS = 250
+
+      class Options
+        attr_accessor :delivery_timeout_millis, :session_id, :perform_checks,
+                      :uuid, :metrics_timeout_millis, :now_millis, :should_apply_treatment,
+                      :view_id, :user_id, :insertion, :platform_id, :client_log_timestamp,
+                      :event_api_timestamp, :request_id, :full_insertion, :use_case, :request
+
+        def initialize()
+        end
+
+        def set_request_params args = {}
+          args = tranlate_args(args)
+          @request                 = args[:request]
+          @delivery_timeout_millis = args[:delivery_timeout_millis] || DELIVERY_TIMEOUT_MILLIS
+          @session_id              = args[:session_id]
+          # @user_id                 = args[:user_id]
```

Comment:
Why are these commented out?

Reply:
Is it because they are set on the userInfo object?

Reply:
Yeah, because we are setting it on userInfo

Reply:
Please change the lines to indicate we are not setting these because these are set on userInfo.